static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -0,0 +1,151 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+import {Observer} from 'mobx-react';
+import * as qs from 'query-string';
+
+import {addLoadingMessage} from 'sentry/actionCreators/indicator';
+import {BaseAvatar} from 'sentry/components/core/avatar/baseAvatar';
+import {Button} from 'sentry/components/core/button';
+import type {SelectKey, SelectOption} from 'sentry/components/core/compactSelect';
+import {CompactSelect} from 'sentry/components/core/compactSelect';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import Form from 'sentry/components/forms/form';
+import FormModel from 'sentry/components/forms/model';
+import List from 'sentry/components/list';
+import ListItem from 'sentry/components/list/listItem';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+type Installation = {
+  avatar_url: string;
+  id: number;
+  login: string;
+};
+
+type GithubInstallationProps = {
+  installation_info: Installation[];
+};
```

Comment:
how does somebody select to install on a new GH account if they have a list of existing installations? or if they select an existing installation and then change their mind and want to select "install on a new GH account"?

Reply:
i think it would be good idea to also have a dropdown option to install on a new GH org, it's not entirely clear what "Skip" means here

Reply:
lemme try this and ill post a pic with update

Reply:
Here's the flow with the dropdown skip. I think it's more clear?

https://github.com/user-attachments/assets/7780e925-4f6f-4019-a377-95a5dc6024e6